language-design.md

```diff
@@ -59,7 +193,34 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
-
+When designing an API, one should have a specific target user-base in mind so
+that one can make design decisions that cater to this user-base. 
+For example, a grayscale function that "also accepts numbers from 0-255" 
+[Verou, 2014].
+This range of numbers would seem completely arbitrary to a beginner programmer.
+However, presumably because the intended users of this API are programmers
+that are familiar with RGB values, this range makes sense.
+Verou even uses this familiarity to argue for naming the function `rgb()`.
+"The benefit is that authors are already accustomed to using rgb() for colors"
+[Verou, 2014].
+
+The goal of naming this function is to pick a name that is intuitive and
+"consistent with author expectations" [Verou, 2014]. 
+But determining which name is most intuitive is difficult.
+The comments seem to indicate that people have very differing naming 
+preferences, no option is a clear winner.
+In fact, many suggest their own option for naming this function.
+It seems that Verou's method of taking a poll is be the best way of determining
+intuitiveness, which is unfortunate as not everybody has the option or means
+to take a poll for every contentious design decision. 
```

Comment:
It's interesting to me as many of the commenters seemed to be trying to get the most perfect name for this function. Maybe there is a huge difference between the names "black" and "white", but for some of the comments, it seemed like arguing about something that 99% would like versus something that a different 99% would like